tests/fixtures/saas_example_fixtures.py

```diff
@@ -658,7 +658,7 @@ def planet_express_invalid_dataset() -> str:
 
 @pytest.fixture
 def planet_express_icon() -> str:
-    return encode_file_contents(
+    return load_as_string(
```

Comment:
A recent changed exposed this mistake in the fixtures for the connector template tests. The SVG should be imported as a string without any encoding.